src/Mod/BIM/Dice3DS/dom3ds.py

```diff
@@ -1717,7 +1717,7 @@ class HIDE_TRACK_TAG(UndefinedChunk):
 class DUMMY(ChunkBase): tag = 0xFFFF
 
 #
-# Obsolute Chunks
+# Obsolete Chunks
```

Comment:
Just making sure this is 'Obsolete' and not 'Absolute'

Reply:
Guess so

Reply:
I agree, it has to be "Obsolete" as it refers to the old undefined chunk after this comment.
It is also a less significant part of the word which tends to get more spelling issues than the start :)